torchtitan/models/llama/__init__.py

```diff
@@ -30,6 +31,9 @@
 
 llama3_configs = {
     "debugmodel": ModelArgs(dim=256, n_layers=2, n_heads=16, rope_theta=500000),
+    "debugmodel_8_layers": ModelArgs(
```

Comment:
I wonder shall we just change the default debug model to 8 layers instead?

Reply:
yea, i can do that.  keep things simpler overall. 

i know it'll affect other PP tests, which need their split-points adjusted.  but that's a really easy change.  I don't know if it would affect any other things.  hopefully not!

Reply:
hmm. now that i made the debug model have more layers, fused rmsnorm throws errors in more cases.  i will have to disable fused_rmsnorm for more pp tests.  something is not right